src/config.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+const path = require('path')
+
+const config = {
+  version: 'v0.4.1', // ipfs dist version
+  defaultExecPath: path.join(__dirname, '..', 'vendor'), // default path for the downloaded exec
+  gracePeriod: 7500 // amount of ms to wait before sigkill
+}
+
+config.baseUrl = 'https://dist.ipfs.io/go-ipfs/' +
+  config.version +
+  '/go-ipfs_' +
+  config.version + '_'
```

Comment:
Unfortunately `path.join` when used on URLs strips down the `//` to `/` and invalidates it. The result would be `https:/dist.ipfs.io/go-ipfs/v0.4.1/go-ipfs_v0.4.1`. I can leave the protocol aside, join it later and join everything else with `path.join` if you think that's worth it.